debian/control

```diff
@@ -18,6 +18,7 @@ Depends: ${python3:Depends},
          fonts-dejavu-core,
          gir1.2-gtk-3.0,
          lzma | xz-utils (>= 5.1.1alpha+20120614-1) | xz-lzma,
+         mdadm (> 4.1~rc1-2),
```

Comment:
I think this should go into recommends.  Otherwise it's going to mean that updated dell-recovery will not be able to install onto older media.